src/less/_variables.less

```diff
@@ -44,11 +44,11 @@
 /* Grayscale */
 @black:                                     #000000;
 @nearblack:                                 #111111;
-@gray-dark:                                 #262626;
+@gray-dark:                                 #222222;
```

Comment:
Hmm, this variable is used as the default `@text-color` variable, which is then used by several other components. Might not be what we want if we only want to change the topbar colors. What do you think?

Reply:
Changed it based on the "new" gray scale that will be implemented in [DG-379](https://payexjira.atlassian.net/secure/RapidBoard.jspa?rapidView=80&projectKey=DG&view=planning&selectedIssue=DG-379). Seemed like the #262626 was not going to be used anymore, but could be a misunderstanding on my part.

Reply:
Ah, there's a new gray scale
Aight :)

👍